api/lib/CominoUtils.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request-promise');
+const { Systems } = require('./Constants');
+
+const fetchCominoDocuments = async (params = {}, token) => {
+  const { id, gateway, ref } = params;
+
+  const requestOptions = {
+    headers: {
+      Authorization: `Bearer ${token}`,
+      'Content-Type': 'application/json'
+    },
+    resolveWithFullResponse: true,
+    uri: `${process.env.W2_DOCUMENTS_API}/${gateway}/customerDocuments/${id}?ref=${ref}`
+  };
+
+  const httpResponse = await request.get(requestOptions);
+
+  return JSON.parse(httpResponse.body);
+};
+
+const processCominoDocuments = (documents, buildDocument) => {
+  return documents.map(doc => {
+    return buildDocument({
+      id: doc.DocNo,
```

Comment:
Wondering if we can tidy up the response from the docs API before we go ahead with this change? I think we're exposing a lot of the DB internals in out data strucutre but I'd rather we used a structure that we define ourselves. e.g. use id instead of DocNo, camelcase not PascalCase, etc. Easier to change now than once this is merged

Reply:
Can add an extra step to `getCustomerDocuments` in [docs API's W2 gateway](https://github.com/LBHackney-IT/w2-document-api/pull/8/files#diff-07e7fce12a0f010889d11ea90a79eb28R35)?

Reply:
Changes in a separate commit here (will tidy before merge): https://github.com/LBHackney-IT/housing-needs-single-view-proto-api/pull/61/commits/716f6fff9b98fc0e1600110c7293c282f49b9d51

Much cleaner already, thanks for the suggestion!

Can go through again if I missed any more clean-up opportunities.